scipy/stats/_continuous_distns.py

```diff
@@ -8910,30 +8918,59 @@ class rv_histogram(rv_continuous):
     """
     _support_mask = rv_continuous._support_mask
 
-    def __init__(self, histogram, *args, **kwargs):
+    def __init__(self, histogram, *args, density=None, **kwargs):
         """
         Create a new distribution using the given histogram
 
         Parameters
         ----------
         histogram : tuple of array_like
-          Tuple containing two array_like objects
-          The first containing the content of n bins
-          The second containing the (n+1) bin boundaries
-          In particular the return value np.histogram is accepted
+          Tuple containing two array_like objects.
+          The first containing the content of n bins,
+          the second containing the (n+1) bin boundaries.
+          In particular, the return value of np.histogram is accepted.
+        density : bool, optional
+          If False, assumes the histogram is proportional to counts per bin;
+          otherwise, assumes it is proportional to a density.
+          For constant bin widths, these are equivalent.
+          For the default np.histogram output, this should be False.
+          If None (default), sets density=True for backwards compatibility,
+          but may give a warning if the bin widths are variable.
```

Comment:
```suggestion
          but warns if the bin widths are variable.
```
I understand that it comes down to the tolerance of `allclose`, but for simplicity, I think it's ok to say "warns". instead of "may warn".